using-timescaledb/backup.md

```diff
@@ -109,151 +109,212 @@ psql -d new_db -c "\COPY conditions FROM data.csv CSV"
  So to speed up importing larger amounts of data, we recommend using
  our [parallel importer][] instead.
 
-## Using Docker & WAL-E [](docker-wale)
-
-If you're using TimescaleDB in a containerized environment, we provide
-images that allow you hook your TimescaleDB container with a WAL-E
-"sidecar" container (i.e., a container that runs alongside the main
-container). Our [official Docker image][docker image] supports this
-functionality, and we provide a reference implementation of the
-[WAL-E sidecar image][wale image].
-
-To get started, you'll first need to start up your Docker TimescaleDB
-database regularly to have it create the necessary directories for
-PostgreSQL:
-```bash
-docker run​ \
-    -v /host/path/to/wal:/data/wal \
-    -v /host/path/to/data:/var/lib/postgresql/data \
-    -e PGWAL=/data/wal \
-    -e PGDATA=/var/lib/postgresql/data/pgdata \
-    ...  # other flags/arguments
-    timescale/timescaledb:latest-pg9.6
-```
->:TIP: You can change the tag from `latest-pg9.6` to the one that best
-suits you, including `latest-pg10` for PostgreSQL 10.
+## Continuous archiving and recovery with Docker & WAL-E [](docker-wale)
 
-The values for `PGWAL` and `PGDATA` are up to you for your setup;
-remember these values since the WAL-E sidecar needs to know them
-(and share access to the directories) in order to coordinate the backups.
+When using TimescaleDB in a containerized environment, it is possible
+to do [continuous archiving][pg archiving] using a [WAL-E][wale
+official] "sidecar" container (i.e., a container that runs alongside
+the main container). For this purpose, we provide a [WAL-E sidecar
+image][wale image] that works with TimescaleDB as well as regular
+PostgreSQL. In the following example, we will setup archiving to the
+local filesystem using one main TimescaleDB container called
+`timescaledb`, and one WAL-E sidecar called `wale`. For production
+deployments, this example can be adapted to do archiving against,
+e.g., AWS S3, and run in an orchestration framework like Kubernetes.
 
-### Backup [](docker-wale-backup)
+### Running the TimescaleDB container
 
-#### TimescaleDB container
+The make TimescaleDB use the WAL-E sidecar for archiving, the two
```

Comment:
Do you think the sidebar entry should change for this section?

Reply:
W.r.t. the sidebar, are you referring to the "Continuous archiving and recovery with Docker & WAL-E"? I think the sidebar can remain "Docker & WAL-E" since it is shorter.